ports/mimxrt/hal/qspi_nor_flash_config.c

```diff
@@ -29,7 +29,11 @@ const flexspi_nor_config_t qspiflash_config = {
     {
         .tag = FLEXSPI_CFG_BLK_TAG,
         .version = FLEXSPI_CFG_BLK_VERSION,
+#if defined(MICROPY_HW_FLASH_DQS)
+		.readSampleClkSrc = MICROPY_HW_FLASH_DQS,
```

Comment:
IMO it's cleaner to have at the top of this file something like:
```c
#ifndef MICROPY_HW_FLASH_DQS
#define MICROPY_HW_FLASH_DQS kFlexSPIReadSampleClk_LoopbackFromDqsPad
#endif
```
Then the code here is simply:
```c
.readSampleClkSrc = MICROPY_HW_FLASH_DQS,
```

Reply:
i agree this way is cleaner